dracut/30coreos-installer/parse-coreos.sh

```diff
@@ -24,6 +24,13 @@ then
     echo $IGNITION_URL >> /tmp/ignition_url
 fi
 
+local CALLHOME_URL=$(getarg coreos.inst.callhome_url=)
+if [ ! -z "$CALLHOME_URL" ]
+then
+    echo "preset call home url to $CALLHOME_URL" >> /tmp/debug
+    echo $CALLHOME_URL >> /tmp/callhome_url
```

Comment:
I'd quote this just to be sure. It can contain `&` which can break in interesting ways.

```suggestion
    echo "$CALLHOME_URL" >> /tmp/callhome_url
```

Reply:
Very nice catch.